src/Saml2/Auth.php

```diff
@@ -264,11 +265,11 @@ public function processResponse($requestId = null)
     /**
      * Process the SAML Logout Response / Logout Request sent by the IdP.
      *
-     * @param bool        $keepLocalSession             When false will destroy the local session, otherwise will keep it
```

Comment:
Why all of the whitespace changes throughout the files?

Reply:
I will fix them and push again.